pkgs/rpm/clapper.spec

```diff
@@ -26,7 +26,7 @@ URL:				https://github.com/Rafostar/clapper
 BuildArchitectures:	noarch
 BuildRoot:			%{_builddir}/%{name}-%{version}-build
 BuildRequires:		meson gjs
-Requires:			gjs gstreamer gstreamer1(element-glsinkbin) gstreamer1(element-gtkglsink)
+Requires:			gjs gstreamer gstreamer-plugins-base gstreamer-plugins-good-gtk
```

Comment:
Ok, this seems specific to RHEL; thats why I didn't have any issues

Reply:
@sp1ritCS 
Also I am not sure since I am not used to packaging like you. Should this be here or inside fedora/rhel requirements? Normally `gtkglsink` comes inside `gstreamer1-plugins-good` (according to GStreamer website), but in Fedora it is in separate package `gstreamer-plugins-good-gtk` (seems to be fedora/rhel specific).

Reply:
`gtkglsink` is in `gstreamer*plugins-good-gtk` for all rpm based distros. Might be in debian or so the case that all of them are in `gstreamer*plugins-good`, but  only the rpm packages provide gstreamer1(element-gtkglsink)()(64bit).

https://pkgs.org/search/?q=gstreamer1(element-gtkglsink)()(64bit)